hackerrank/src/Algorithms/Strings/CamelCase.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Fedotkin.Dotnet.HackerRank.Algorithms.Strings.CamelCase;
+
+public class Result
+{
+    public static int CamelCase(string s)
+    {
+        if (string.IsNullOrEmpty(s))
+            return 0;
+
+        int result = 1;
+
+        foreach (char symbol in s)
+        {
+            if (char.IsUpper(symbol))
+                result++;
+        }
```

Comment:
```suggestion
        int result = s.Count(ch => char.IsUpper(ch)); // LINQ to Objects
```
або нават так
```csharp
int result = s.Count(char.IsUpper);
```
Патлумачыць?